src/Legacy/Storage.php

```diff
@@ -888,8 +888,9 @@ public function getContentByTaxonomy($taxonomyslug, $name, $parameters = [])
 
         $slug = $this->app['slugify']->slugify($name);
 
-        $limit = $parameters['limit'] ?: 9999;
-        $page = $parameters['page'] ?: 1;
+        $limit = (!empty($parameters['limit']) && $parameters['limit']) ? $parameters['limit'] : 9999;
+        $page = (!empty($parameters['page']) && $parameters['page']) ? $parameters['page'] : 1;
```

Comment:
I think `empty` is pretty much a codesmell, could this not be changed to `$page = isset($parameters['page']) ? $parameters['page'] : 1;` and pretty much the same for the above line?

When we move to php7 only we can also switch this to `$page = $parameters['page'] ?? 1;`

Also, the newline below is unnecessary.

Reply:
I'm not sure if we should allow limit === 0 pass. Because the original code was using the second ternary block in case of zero limit.  
So, probably we should change `!empty` to `isset`, but leave the second condition?

p.s. love that `??` php7 feature :(